noetic/distribution.yaml

```diff
@@ -2213,6 +2213,21 @@ repositories:
       url: https://github.com/mikeferguson/graceful_controller.git
       version: ros1
     status: developed
+  graft:
+    doc:
+      type: git
+      url: https://github.com/ros-perception/graft.git
+      version: hydro-devel
```

Comment:
Is it really the case that the `hydro-devel` still builds/works on Noetic?  I guess it is not impossible, it would just be surprising.

Reply:
It compiles and works - there are basically no outside dependencies in this package

Reply:
Heh, OK.  Thanks.